roadmap/implementers-guide/src/runtime/inclusion.md

```diff
@@ -69,7 +69,7 @@ All failed checks should lead to an unrecoverable error making the block invalid
   1. Check the collator's signature on the candidate data.
   1. check the backing of the candidate using the signatures and the bitfields, comparing against the validators assigned to the groups, fetched with the `group_validators` lookup.
   1. call `Router::check_upward_messages(para, commitments.upward_messages)` to check that the upward messages are valid.
-  1. call `Router::check_processed_downward_messages(para, commitments.processed_downward_messages)` to check that the DMQ is properly drained.
+  1. call `Router::check_dmq_watermark(para, commitments.dmq_watermark)` for each canddiate to check rules of processing the DMQ watermark.
```

Comment:
```suggestion
  1. call `Router::check_dmq_watermark(para, commitments.dmq_watermark)` for each candidate to check rules of processing the DMQ watermark.
```

Reply:
The sentence here is not entirely clear to me, are the messages drained down to the watermark?

Reply:
No, this call is part of the `process_candidates` invocation and that is happening at the inclusion time. That implies that we only checking the validity of the candidates at this time and we are not enacting any changes by this candidate as of yet.